src/pages/Home.jsx

```diff
@@ -1,10 +1,123 @@
 import Navbar from "@/components/Navbar";
+import DocFileIcon from "@/assets/home_doc_file_icon.svg";
+import DocSortingIcon from "@/assets/home_doc_sorting_icon.svg";
+// import DocSortingExpandIcon from "@/assets/home_doc_sorting_expand_icon.svg";
+// import Typography from '@mui/material/Typography';
+// import Popover from '@mui/material/Popover';
+// import { useState } from "react";
+
+
+// const SortingDropDownMenu = (props) => {
+//   return <div className={'flex flex-col bg-red-100 w-[100px] border-solid rounded ' + props.className}>
+//     <div>1</div>
+//     <div>2</div>
+//     <div>3</div>
+//   </div>
+// }
+
+// const DocumentsSelectionTopBar = () => {
+//   const [anchorEl, setAnchorEl] = useState(null);
+
+//   const handlePopoverOpen = (event) => {
+//     setAnchorEl(event.currentTarget);
+//   };
+
+//   const handlePopoverClose = () => {
+//     setAnchorEl(null);
+//   };
+
+//   const open = Boolean(anchorEl);
+
+//   return <div className='pt-5'>
+//     <div className='text-xl font-bold'>
+//       <div className="flex flex-row"> 
+//         <div>Documents</div>
+//         <img src={DocSortingIcon} className="ml-5"/>
+//         <Typography
+//           aria-owns={open ? 'mouse-over-popover' : undefined}
+//           aria-haspopup="true"
+//           onMouseEnter={handlePopoverOpen}
+//           // onMouseLeave={handlePopoverClose}
+//         >
+//           <img src={DocSortingExpandIcon}/>
+//         </Typography>
+//       </div>
+//       <Popover
+//         id="mouse-over-popover"
+//         sx={{
+//           pointerEvents: 'none',
+//         }}
+//         open={open}
+//         anchorEl={anchorEl}
+//         anchorOrigin={{
+//           vertical: 'bottom',
+//           horizontal: 'left',
+//         }}
+//         transformOrigin={{
+//           vertical: 'top',
+//           horizontal: 'left',
+//         }}
+//         onClose={handlePopoverClose}
+//         onMouseLeave={handlePopoverClose}
+//         disableRestoreFocus
+//       >
+//         <SortingDropDownMenu/>
+//       </Popover>
+//     </div>
+//   </div>
+// }
+
+const DocumentsSelectionTopBar = () => {
+
+  return <div className='pt-5'>
+    <div className='text-xl font-bold'>
+      <div className="flex flex-row"> 
+        <div>Documents</div>
+        <img src={DocSortingIcon} className="ml-5"/>
+      </div>
+    </div>
+  </div>
+}
+
+const DocumentFileText = ({doc}) => {
```

Comment:
What's the doc variable supposed to be here? the document title or the body?

Reply:
Doc is everything but the body, as in everything in the first part of the data model. But it was a placeholder until I got the format of the doc
<img width="279" alt="Screenshot 2023-11-22 at 6 30 59 PM" src="https://github.com/wendi-yu/newbloom/assets/46700318/cecdf1e7-381d-4efd-bf7a-0c8a56274c8e">